README.md

```diff
@@ -114,7 +114,7 @@ fastify.register(require('@fastify/auth'), { defaultRelation: 'and'} )
 
 _For more examples, please check [`example-composited.js`](test/example-composited.js)_
 
-This plugin support `callback` and `Promise` returned by the functions. Note that an `async` function **does not have** to call the `done` parameter, otherwise the route handler to which the auth methods are linked to [might be called multiple times](https://www.fastify.io/docs/latest/Hooks/#respond-to-a-request-from-a-hook):
+This plugin support `callback` and `Promise` returned by the functions. Note that an `async` function **does not have** to call the `done` parameter, otherwise the route handler to which the auth methods are linked to [might be called multiple times](https://www.fastify.dev/docs/latest/Hooks/#respond-to-a-request-from-a-hook):
```

Comment:
page not found :/

Reply:
```suggestion
This plugin support `callback` and `Promise` returned by the functions. Note that an `async` function **does not have** to call the `done` parameter, otherwise the route handler to which the auth methods are linked to [might be called multiple times](https://fastify.dev/docs/latest/Reference/Hooks/#respond-to-a-request-from-a-hook):
```

Reply:
> page not found :/

Good spot, thought I'd checked that one 🙃

Reply:
Well, tbh. I was like: "man I should review properly, not that I miss something out and then the questions arises if my PR reviews are proper or not, lets start with that one. Oh 404" :D

Well, you accepted the suggestion fast than I could fix my i accidental removal of the `www.`. but the link is still working.

Should we fix that too?

Reply:
Could probably remove the `www.` from all the URLs in docs and save a few bytes.
I'll add it to my list; planning to do a batch of PRs tomorrow.

Reply:
Allrighty, friend ;).